src/test/resources/features/TalkModule.feature

```diff
@@ -1,5 +1,47 @@
-@wip
-Feature: Talk Module
 
-  Background:
-    Given the user is logged in
\ No newline at end of file
+Feature: Dashboard Functionality
+
+  Background: As a user I should be able to modify Dashboard page
+    Given the user is logged in
+
+
+  Scenario: User can see all modules and Username after login (1)
+    When User enters a valid username
+    And User enters valid password and clicks the Login button
+    Then User successfully login on the dashboard page and see all modules
+
+  Scenario: User can click on Customize button and see the status widgets (2.1)
+    When User clicks on Customize button
+    And  User can see the status widgets
+    Then User can select any of status widges
```

Comment:
i dont understand this code. it may be due to  that i  dont see the related part in UI